File: Core/Entities/Product.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Core.Entities;

public class Product: BaseEntity
{
    public required string Naam { get; set;}
    public required string Beschrijving {get; set;}
    public decimal Prijs {get; set;}
    public required string FotoURL {get; set;}
    public required string Type {get; set;}
    public required string Merk {get; set;}
    public int HoeveelheidInVoorraad {get; set;}
}

File: client/src/app/shared/models/product.ts
export interface Product {
    id: number;
    naam: string;
    beschrijving: string;
    prijs: number;
    fotoURL: string;
    type: string;
    merk: string;
    hoeveelheidInVoorraad: number;
}


export interface NewProduct {
    id?: number;
    naam: string;
    prijs: number;
    merk: string;
    type: string;
    hoeveelheidInVoorraad: number;
    beschrijving: string;
    fotoURL?: string;
    image?: File;
  }